### DISCUSSION TITLE
Can't get PDF files to show in Chromium ver 115.3.130

### DISCUSSION THREAD

**Discussion by BillBogart** [cefsharp/CefSharp#4904] [General]
We upgraded our version of chromium last year from from ver 63.0.3 to ver 115.3.130. It is integrated into our product and now it no longer displays pdf files.  It only shows a dark screen where the pdf viewer should show the pdf.  I have ran CefSharp.MinimalExample.WinForms.net472 ver 115.3.130 on the same PC and it displays pdfs without issues.   I have looked through verbose log files from both comparing them.  

I attached the full log file of the failed PDF view.  

But here is a partial log file:
[0809/151933.267:VERBOSE1:frame_host_impl.cc(503)] frame 30064771089 [7,17] (sub) detached (reason=RENDER_FRAME_DELETED, is_connected=0)
[0809/152032.239:VERBOSE1:script_context.cc(143)] Created context:
  extension id:           (none)
  frame:                  24D29608
  URL:                    
  context_type:           UNSPECIFIED
  effective extension id: (none)
  effective context type: UNSPECIFIED
[0809/152032.240:VERBOSE1:script_context.cc(143)] Created context:
  extension id:           (none)
  frame:                  00000000
  URL:                    
  context_type:           UNSPECIFIED
  effective extension id: (none)
  effective context type: UNSPECIFIED
[0809/152032.240:VERBOSE1:dispatcher.cc(456)] Num tracked contexts: 1
[chrome_debug_pdf.log](https://github.com/user-attachments/files/16565993/chrome_debug_pdf.log)

**Comment by amaitland** [comment]
Does your application have the appropriate app.manifest?

https://github.com/cefsharp/CefSharp/wiki/Quick-Start-For-MS-.Net-4.x#40-add-appmanifest-to-your-application

If that doesn't help try upgrading to the current supported version.

**Reply by BillBogart** [reply]
Amaitland, thanks.  I did figure out the issue.  There was code in OnBeforeBrowse() that was trying to evaluate and disallow protocols and was returning true when loading a pdf and or a chrome extension.  I changed it to return false. when loading a pdf and or chrome: extension.
